data_validation/src/spark_rapids_validation_tool/dataproc_wrapper.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrapper class to run tools associated with RAPIDS Accelerator for Apache Spark plugin."""
+import fire
+from spark_rapids_validation_tool.data_validation_dataproc import DataValidationDataproc
+
+class DataprocWrapper(object):
+
+    def validation(self,
+                   cluster: str,
+                   region: str,
+                   check: str = 'valid_metadata',
+                   format: str = None,
+                   t1: str = None,
+                   t1p: str = None,
+                   t2: str = None,
+                   t2p: str = None,
+                   pk: str = None,
+                   e: str = None,
+                   i: str = 'all',
+                   f: str = None,
+                   o: str = None,
+                   of: str = 'parquet',
+                   p: int = 4,
+                   debug: bool = False) -> None:
```

Comment:
Could you please update the arg names so that it's easy to understand/read. Would be good to change it in all places including commandline.

Reply:
updated all arg&params